### DISCUSSION TITLE
npm run generate Not Working During Migration to 0210

### DISCUSSION THREAD

**Discussion by vvs-basanth** [StellaX-Technologies/gaac-hub#3] [General]
An issue while trying to migrate my project to version 0210. 
When running the `npm run generate` command, it doesn't work as expected.

**Steps to Reproduce:**

1. **Fork and Clone the repository:** (If applicable) Briefly describe how you obtained the code (e.g., cloned the repo, downloaded a specific version).
2. **Install dependencies:** Mention any specific installation steps required, especially if they differ from the standard `npm install`.
3. **Attempt migration:** Explain how you tried to migrate to version 0210 (e.g., specific commands used, following migration guide).
4. **Run `npm run generate`:** Describe how you executed the command and in what context (e.g., terminal, specific directory).

**Additional Context:**

* **Version Information:** Mention the specific versions of the project (if applicable), Node.js, and npm you're using.
* **Code Snippets:** If relevant, share small code snippets that demonstrate the issue (be mindful of sensitive information).
* **Configuration Files:** If the issue might be related to configuration files, provide relevant excerpts (redacted if necessary).

**Desired Outcome:**

State what you'd like to achieve by reporting this issue. Do you want help troubleshooting, suggestions for an updated `generate` command, or a potential bug fix?

**Thank you for your time and assistance!**

**Additional Tips:**

* Check the project's documentation or migration guide for any specific instructions related to version 0210.
* Search for existing issues related to the `generate` command or migration process.
* If you have a workaround or potential solution, feel free to share it as well.

**Comment by SKParthiv** [comment]
Issue Solved

Steps done to Reproduce:
1) Fork and Clone Repo: Using GitHub Desktop
2) Used `npm install` (Not related to issue but used `npm update` which updated yaml version 2.4.3 to 2.4.5
3) Run `npm run generate`
 
Actual Outcome: 
 Error:  This command is deprecated, please use updated 'generate' command (see https://orm.drizzle.team/kit-docs/upgrade-21#how-to-migrate-to-0210)
 
  
 Desired Outcome:
 Error: Cannot find the url
 
 Steps done to solve:
 1) Followed link https://orm.drizzle.team/kit-docs/upgrade-21#how-to-migrate-to-0210
 2) Removed all the `:dialect` in all scripts in package.json
 3) Added a param `dialect: "postgresql"` and changed `connectionString` param to `url` param in drizzle.config.ts
 4) Run `npm run generate`
 
Final Outcome:
Error: Cannot find the url
